deploy/compute/build-upload.sh

```diff
@@ -3,5 +3,5 @@
 JAR_TAG=${1:-dev}
 
 sbt assembly && \
-gsutil cp target/scala-2.12/constellation-assembly-1.0.1.jar gs://constellation-dag/release/dag-$JAR_TAG.jar && \
+gsutil cp target/scala-2.12/constellation-assembly-1.0.10.jar gs://constellation-dag/release/dag-$JAR_TAG.jar && \
```

Comment:
this could be automated by extracting version to version.sbt
```suggestion
version="$(cat  version.sbt  | cut -d '-' -f 1 | cut -d '"' -f 2)"
gsutil cp target/scala-2.12/constellation-assembly-$versionjar gs://constellation-dag/release/dag-$JAR_TAG.jar && \
```
or just use 
```
sbt 'inspect actual version' | grep "Setting: java.lang.String" | cut -d '=' -f2 | tr -d ' '
```

Reply:
Nice find, will add